### ISSUE TITLE
Improve password security

### ISSUE THREAD

**Issue by ghost** [bmilliron/OpenRPSDev#2]
Using simply md5 for a password is very insecure. Those kind of passwords cracked in seconds thanks to the multiple rainbow tables online.

I recommend using salted hashes to add some randomness to it.

**Comment by irontet** [comment]
I agree

**Comment by ghost** [comment]
@JackoThe1st, Do you know how to do this? I'm currently busy on login and registration.

**Comment by ghost** [comment]
Not off the top of my head, but I can give it a go once I've figured out how to set up the database locally on my machine.

**Comment by irontet** [comment]
You can point to the dev database, I will PM you the details on the forums. NOTE: We moved the repo now to its own organization here, so clone this one: https://github.com/OpenRPS/OpenRPSDev

**Comment by ghost** [comment]
@JackoThe1st is setting up the database going well?

**Comment by ghost** [comment]
Nearly there. I decided to ditch MAMP and set up my web stack individually for the first time. Database and tables are ready to go, I just need to connect to it. Did you manage to get Postgres working alongside WAMP?

**Comment by irontet** [comment]
Jack, Don't forget to use the new repo. The bmilliron/OpenRPS repo is obsolete.
The github link on the openrps.org web site is the current master repo. Thanks,
ST

On Sun, Jul 5, 2015 at 6:55 PM, JackoThe1st notifications@github.com wrote:
Nearly there. I decided to ditch MAMP and set up my web stack individually for
the first time. Database and tables are ready to go, I just need to connect to
it. Did you manage to get Postgres working alongside WAMP?

—
Reply to this email directly or view it on GitHub
[https://github.com/bmilliron/OpenRPSDev/issues/2#issuecomment-118674674] .[https://github.com/notifications/beacon/AJkWn4GQcAbCbEWnuUYuoRndMLByP3ppks5oaa1NgaJpZM4FPNCF.gif]

**Comment by ghost** [comment]
@JackoThe1st Yes i'm running postgres alongside WAMP. I had the server already installed from another time. I had to load postgres in apache and PHP.

**Comment by zochamx** [comment]
I can (and would like to) help with the password bug. May i ?

**Comment by Vincentdc94** [comment]
@tilemachoz We moved to laravel a long time ago. Check http://openrps.org/forums/ if you want to join in development

**Comment by zochamx** [comment]
hmm didn't find it anywhere thanks i will check it out.
